crates/aranya-daemon-api/src/service.rs

```diff
@@ -44,36 +57,71 @@ impl fmt::Display for Error {
 
 impl core::error::Error for Error {}
 
+/// Result type used throughout the daemon API.
+///
+/// This type alias simplifies error handling by using the API's [`Error`] type
+/// as the default error type.
 pub type Result<T, E = Error> = core::result::Result<T, E>;
 
 custom_id! {
     /// The Device ID.
+    ///
+    /// A unique identifier for an Aranya device within the system.
+    /// This ID is used to reference specific devices when performing operations
+    /// such as adding devices to teams or assigning roles.
     pub struct DeviceId;
 }
 
 custom_id! {
     /// The Team ID (a.k.a Graph ID).
+    ///
+    /// A unique identifier for a team of devices in Aranya.
+    /// Teams represent a group of devices that can collaborate and communicate
+    /// with each other securely.
     pub struct TeamId;
 }
 
 /// A device's public key bundle.
+///
+/// This structure contains the public keys needed for cryptographic operations
+/// when communicating with a device, including identity verification, signature
+/// verification, message encryption, and data integrity.
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct KeyBundle {
+    /// The identity public key used to verify the device's identity
     pub identity: Vec<u8>,
+
+    /// The signing public key used to verify the device's signature
     pub signing: Vec<u8>,
-    pub encoding: Vec<u8>,
+
+    /// The encryption public key used for message encryption
+    pub encryption: Vec<u8>,
 }
 
 /// A device's role on the team.
+///
+/// Roles determine what permissions a device has within a team.
+/// Different roles have different capabilities regarding team management
+/// and access control. For a more detailed permissions breakdown, see policy.md.
```

Comment:
```suggestion
/// and access control. For a more detailed permissions breakdown, see `aranya-daemon/src/policy.md`.
```
Not sure if it's better to do this or just `policy.md`